ecommerce/extensions/payment/processors/stripe.py

```diff
@@ -103,6 +103,12 @@ def generate_basket_pi_idempotency_key(self, basket):
     def get_capture_context(self, request):
         # TODO: consider whether the basket should be passed in from MFE, not retrieved from Oscar
         basket = Basket.get_basket(request.user, request.site)
+        if basket.lines.exists():
```

Comment:
Should it be the other way around?

```suggestion
        if not basket.lines.exists():
```

Reply:
Yep! Updated.